src/routes/add-place.ts

```diff
@@ -33,22 +37,24 @@ export default async function addPlace(fastify: FastifyInstance) {
       hierarchy: Config.getHierarchyWithReplacement(contactType, 'desc'),
       contactType,
       contactTypes,
-      userRoleProperty: Config.getUserRoleConfig(contactType)
+      userRoleProperty: Config.getUserRoleConfig(contactType),
     };
 
-    return resp.view('src/liquid/app/view.html', tmplData);
+    return resp.view('src/liquid/app/view.liquid', tmplData);
   });
 
   fastify.post('/place/dob', async (req, resp) => {
     const { place_type, prefix, prop_type } = req.query as any;
-    const contactType = Config.getContactType(place_type).contact_properties.find(prop => prop.type === prop_type);
-    return resp.view('src/liquid/components/contact_type_property.html', {
+    const contactType = Config.getContactType(
```

Comment:
is this clearer? i don't personally find this more readable.  prefer something like:

```
const contactType = Config.getContactType(place_type)
  .contact_properties
  .find(prop => prop.type === prop_type);
```

Reply:
Yes, this should be the [coding style](https://docs.communityhealthtoolkit.org/community/contributing/code/style-guide/#method-chaining).

Reply:
Will update, vscode auto-formated the files